### DISCUSSION TITLE
[video] Sneak peek: Provider-defined functions

### DISCUSSION THREAD

**Discussion by ghost** [opentofu/opentofu#1333] [Announcements]
Curious? [Here's the short.](https://youtube.com/shorts/4aHZjDz2VWg)

**Comment by apamildner** [comment]
Really cool! Is there any more info about this feature/when is it likely to be released?

**Reply by ghost** [reply]
Hi @apamildner this feature is currently planned for OpenTofu 1.8. We'll publish more info both on the blog and YouTube when we get closer to the release date.
